Dockerfile

```diff
@@ -1,3 +1,7 @@
+#*********************************************************************
```

Comment:
Add `LABEL` as well -- see https://github.com/open-amt-cloud-toolkit/sample-web-ui/blob/master/Dockerfile

Reply:
Check out my suggestion :) You can even commit it!

Reply:
agreed!  I like it!